dags/common/cleaning/clean_whitespace_characters.py

```diff
@@ -0,0 +1,2 @@
+def clean_whitespace_characters(input):
+    return ' '.join(input.split())
```

Comment:
I don't understand how this works with multiple spaces ? Won't this add spaces between spaces ?

For curiosity purposes, any idea on the performance difference with 
```python
re.sub("\\s+", " ", input)
```
?

Reply:
re.sub("\\s+", " ", input) - won't remove the trailing spaces. The expression you suggested will take 1 or more spaces and replace them with one space, but will leave the ones in the beginning and end of the string

Reply:
Oh yes good catch !